plugins/jira/skills/gcp-hcp/SKILL.md

```
@@ -80,78 +80,34 @@ The GCP project uses these components for organizing work:
-    additional_fields={
-        "customfield_10014": "GCP-456",  # Epic Link - parent epic
-        "customfield_10028": 3.0,        # Story Points - auto-estimated per Sizing Guide
-        "priority": {"name": "Major"},      # Priority - OMIT unless user specifies
-        "labels": ["ai-generated-jira"],
-        "security": {"name": "Red Hat Employee"}
-    }
-)
-```
+**Note:** Always include `contentFormat: "markdown"` when calling `createJiraIssue` or `editJiraIssue` so descriptions are interpreted as Markdown.
+
+#### Custom Fields by Issue Type
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)